copilot/webapp/manifest.yml

```diff
@@ -111,5 +115,6 @@ environments:
         - "give-or-refuse-consent-for-vaccinations.nhs.uk"
     variables:
       RAILS_ENV: production
+      SENTRY_ENVIRONMENT: production
```

Comment:
Not technically needed as Sentry will take from `RAILS_ENV`, but left here for consistency with the others.